pom.xml

```diff
@@ -64,19 +63,17 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
-            <version>2.8.9</version>
+            <version>2.25.1</version>
```

Comment:
is it different from parent-pom?

Reply:
Good question, should probably just have removed it instead of bumping it.

Reply:
Yes please remove Mockito & Powermock versions as in https://github.com/jenkinsci/pipeline-cloudwatch-logs-plugin/pull/47 for example.

Reply:
Done.